torch/serialization.py

```diff
@@ -228,8 +251,12 @@ def load(f, map_location=None, pickle_module=pickle):
         >>> torch.load('tensors.pt')
         # Load all tensors onto the CPU
         >>> torch.load('tensors.pt', map_location=lambda storage, loc: storage)
+        # Load all tensors onto the current GPU (GPU 0 unless torch.cuda.set_device
+        # has been called)
```

Comment:
Please don't mention `torch.cuda.set_device`. We discourage using it. Saying that it will be put on a current GPU is fine. Alternatively, you can make it `storage.cuda(0)`